modules/gapi/src/api/kernels_core.cpp

```diff
@@ -355,5 +355,11 @@ GMat sqrt(const GMat& src)
     return core::GSqrt::on(src);
 }
 
+GMat normalize(const GMat& _src, double a, double b,
+                    int norm_type, int ddepth)
```

Comment:
Alignment was not fixed

Reply:
What do you mean? normalize() itself should remain in lower case